httpclient5/src/main/java/org/apache/hc/client5/http/protocol/RequestIfRange.java

```diff
@@ -121,36 +113,21 @@ public void process(final HttpRequest request, final EntityDetails entity, final
             throw new ProtocolException("'If-Range' header must not contain a weak entity tag.");
         }
 
-        final Header dateHeader = request.getFirstHeader(HttpHeaders.DATE);
+        final Instant dateHeader = DateUtils.parseStandardDate(request, HttpHeaders.DATE);
 
         if (dateHeader == null) {
             return;
         }
 
+        final Instant lastModifiedHeader = DateUtils.parseStandardDate(request, HttpHeaders.LAST_MODIFIED);
```

Comment:
@arturobernalg `lastModifiedHeader` is likely not the best name now. `lastModified`?

Reply:
@ok2c Agree.
Changed.